test/facade_converter_test.ts

```diff
@@ -107,6 +123,13 @@ ByteBuffer buff;`);
 
 var y = x is Promise;`);
     });
+
+    it('works with a different typings directory', () => {
+      expectWithTypingsRootTypes('var y: Promise = Promise.resolve(1);')
```

Comment:
could it just `import {Promise} from 'somewhere'`
and add a somewhere.d.ts in the existing getSources fixture above?